app/main.py

```diff
@@ -1 +1,46 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    use_params(make_params(command))
+
+
+def make_params(command: str) -> list:
+    try:
+        command = command.split(" ")
```

Comment:
`" "` is default value for `command.split`
```suggestion
        command = command.split()
```